content/2022-06-03-brigade-2.6.0-and-friends.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Brigade 2.6.0 and Friends"
+slug: "2022-06-03-brigade-2.6.0-and-friends"
+description: "Brigade 2.6.0 has been released"
+date: "2022-06-03"
+author: "Kent Rancourt"
+summary: "Brigade 2.6.0 and several new peripherals have been released!"
+---
+
+Hello, Brigadiers! We hope everyone's had a great week! It's certainly been a
+great one for us because we're closing it out with the release of Brigade v2.6.0
+and a few new peripherals!
+
+Brigade v2.6.0 continues the trend of small, frequent releases that are more
+evolutionary than revolutionary. Apart from the bug fixes, doc improvements, and
+other maintenance that go into every release, there's just one noteworthy change
+in v2.6.0 -- Brigade is now allowing longer label and qualifier values on events
+and in projects' event subscriptions. As small a change as this may seem, it's a
+key enabler for an important new peripheral...
+
+Today, we're announcing that the
+[Brigade CloudEvents Gateway](https://github.com/brigadecore/brigade-cloudevents-gateway)
+has reached general availability with its v1.0.0 release. Support for
+CloudEvents is _monumentally_ important because of its broadly useful potential
+for integrating Brigade with innumerable third-party services that also act as
+CloudEvent producers.
+
+To illustrate this new gateway's utility, let's consider Microsoft Azure.
+[Azure Event Grid](https://docs.microsoft.com/en-us/azure/event-grid/overview)
+can subscribe to events originating from many, if not most, types of Azure
+resources, transform them to CloudEvents, and deliver those over HTTPS to any
+CloudEvents consumer -- such as the Brigade CloudEvents gateway. And just like
+that -- Brigade v2.6.0+ scripts can be triggered by just about anything in
+Azure. And Azure is just the tip of the iceberg since the CloudEvents 1.0
+specification is being so widely adopted throughout the cloud-native community.
```

Comment:
modifying to avoid saying Azure twice in a row:

```suggestion
Since the CloudEvents 1.0 specification is being so widely adopted throughout the cloud-native community, this peripheral unlocks a wide range of use-cases.
```

Reply:
Rewording with something quite similar to what you have proposed.

Reply:
> And
this example is but the tip of the iceberg. Because the CloudEvents 1.0
specification is being so widely adopted throughout the cloud-native community,
this gateway unlocks a wide range of use cases.